### PR TITLE
ETC Course: 16. What Is WETC?

### PR DESCRIPTION
**Pull request by DonaldMcIntyre**
This is the ETC course class 16 about what is WETC. 

Chinese version done.

It will be published on 2023-04-06.

### CODE REVIEW

**Review by brolal — DISMISSED** [review]

**Review by gitr0n1n — DISMISSED** [review]
Reading this article and the verbiage is very confusing, even for someone that knows what these things are. I think good common practice when writing something like this is to refer to the native asset as "ETC asset" and the network as "the Ethereum Classic network". In this article it hops around back and forth and is hard to follow in English. Probably will have similar issues when translated.

Additionally, @DonaldMcIntyre could you use an asset like a stablecoin over a centralized memecoin clone like "Shiba Classic" in the examples? ETC/stable will be the largest market on the network and makes much more sense than an obscure coin like that.

After reading this, it might be a good idea to reframe this article as "What are wrapped tokens?" since you can speak to various wrapping usecases, opposed to WETC only. Here is a simple writeup of the WETC asset: https://wrappedether.org/

+ Native Wrapped Asset: WETC = Wrapping the native Ether asset to add ERC-20 functionality to it. Since this asset was created prior to the ERC-20 standard existing. You see this with any of the other EVMs e.g.: WETH, WBNB. These assets are typically self-custody and hold the properties of the native network. However, its not always guaranteed the contract is non-custodial. Just as ERC-20's like USDC have a freeze function, a wrapped token could as well. Typically no risk of depegging with one-way order flow, since 100% of assets in contract and readily available to wrap/unwrap, however does cost gas to wrap/unwrap so transaction fee is applicable.

+ Bridged Non-EVM Assets:
++Blockchain asset: WBTC =  Wrapped Bitcoin stored in a custody wallet by a centralized organization. The asset is then minted to an EVM chain where it can interact with other ERC-20's in smart contract financial products. Centralized risk due to the custodian. Risk of depegging with one way order flow. Reference renBTC during FTX.
++Government Issued Asset: USDC = USD stored in various centralized bank accounts. Can be thought of as WUSD. Able to freeze asset with admin keys. Also risk of underlying collateral disappearing in a bank run, reference to Signature Bank and SVB collapsing and part of USDC reserves compromised. Risk of depeg and custody.
++Real World Asset like Precious Metals: DGX = Paper Gold. Can be thought of as WGOLD. Able to freeze asset with admin keys. Also risk of underlying collateral disappearing due to paper gold not existing, misreported, seized. Risk of depeg and custody.

+ Bridged EVM Native ERC-20 Assets:  Cross-chain interoperability via routers like Multichain. Allows EVM assets to be traded across multiple EVMs, but the risk here is the method of bridging. Most bridging is very centralized and if compromised, results in a hack: reference Harmony bridge was 100% centralized, hacked due to private keys stored on a cloud server. Multichain claims a decentralized bridge, but we saw them freeze chains with FTX- so it is clearly a centralized bridge that has custodian risk.

+ EVMs as wrapping network environments: Moonbeam = Polkadot EVM. Kava = Cosmos EVM.

**Comment by DonaldMcIntyre** [comment]
The article is about WETC because it is the ETC course. The article is fine as it is.

File: content/blog/2023-04-06-ethereum-classic-course-16-what-is-wetc/index.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+## WETC Works With All Dapps That Support The ERC-20 Standard
+
+![WETC and dapps.](./4.png)
+
+As so many dapps support the ERC-20 smart contract standard, then WETC has become a convenient way to keep some money to use them.
+
+This is especially true for swap decentralized exchanges where users may trade $WETC for many other kinds of ERC-20 tokens.
+
+In the future many external assets, such as stocks, bonds, mutuals funds, ETFs, derivatives, and futures contracts will be represented as ERC-20 tokens inside blockchains and will be traded in decentralized swap exchanges.
```
Comment (TheCrowbill) [inline content/blog/2023-04-06-ethereum-classic-course-16-what-is-wetc/index.md:67]: Line 67: Correct comma placement; Correct plural
```suggestion
In the future, many external assets such as stocks, bonds, mutual funds, ETFs, derivatives, and futures contracts will be represented as ERC-20 tokens inside blockchains and will be traded in decentralized swap exchanges.
```

File: content/blog/2023-04-06-ethereum-classic-course-16-what-is-wetc/index.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+![WETC in several blockchains.](./5.png)
+
+As ERC-20 tokens may represent assets outside blockchains, then even coins from other blockchains may be represented as ERC-20 tokens.
+
+For example, WETC may exist inside Ethereum Classic to use with dapps, but also inside Ethereum, Binance Smart Chain, and many other smart contracts blockchains.
+
+This facilitates what is called blockchain interoperability.
+
+However, for now the technology that is called “bridges” which does these connections, is not as secure as the blockchain environment, so it is always safer to return back any wrapped tokens to their original chains whenever possible.
```
Comment (TheCrowbill) [inline content/blog/2023-04-06-ethereum-classic-course-16-what-is-wetc/index.md:79]: Line 79: Rewrite for clarity; Correct awkward wording
```suggestion
For now, the technology that handles the connections between different blockchain networks, known as "bridges", is not as secure as the blockchain environments they connect, Therefore, it is safer to return wrapped tokens to their original chains whenever possible.
```

File: content/blog/2023-04-06-ethereum-classic-course-16-what-is-wetc/index.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+Whenever users wish to transform their $WETC back into $ETC, they can send their $WETC to the smart contract and it will return $ETC to their account.
+
+For example, if a user sends 100 $WETC to the Wrapped ETC smart contract, then it will return 100 $ETC to the user’s account.
+
+## WETC Works With All Dapps That Support The ERC-20 Standard
+
+![WETC and dapps.](./4.png)
+
+As so many dapps support the ERC-20 smart contract standard, then WETC has become a convenient way to keep some money to use them.
```
Comment (TheCrowbill) [inline content/blog/2023-04-06-ethereum-classic-course-16-what-is-wetc/index.md:63]: Line 63: Rewrite for clarity
```suggestion
With the majority of dapps on the ETC network supporting the ERC-20 token standard, WETC has since become a convenient way to use and interact with them.
```

**Review by TheCrowbill — CHANGES_REQUESTED** [review]

**Review by TheCrowbill — APPROVED** [review]
All good

**Review by gitr0n1n — APPROVED** [review]
